docs/docs/getting-started/lifecycle-management.md

```diff
@@ -100,7 +100,7 @@ Apply this manifest:
 
 ```yaml
```

Comment:
since we are touching this file I would suggest migrating yamls in separate files

Reply:
we have a special ticket for that https://github.com/keptn/lifecycle-toolkit/issues/2587 I would say I won't move the code blocks to separate files unless I directly touch them

Reply:
but you just did 😢

Reply:
I adapted those that I touched